contracts/src/external/HyperdriveTarget0.sol

```diff
@@ -442,6 +448,15 @@ abstract contract HyperdriveTarget0 is
         _revert(abi.encode(_convertToShares(_baseAmount)));
     }
 
+    /// @notice Gets the total amount of vault shares held by Hyperdrive.
+    /// @dev This is a convenience method that allows developers to get the
+    ///      total amount of vault shares without knowing the specifics of the
+    ///      integration.
+    /// @return The total amount of vault shares held by Hyperdrive.
+    function totalShares() external view returns (uint256) {
```

Comment:
out of curiousity, why isn't there any accompanying `_totalBase()`/`totalBase()` ?

Reply:
We don’t need it for sweep (which is what this is used for), and it should always be 0 since 100% of the funds are allocated to the yield source